app/components/swiper.js

```diff
@@ -0,0 +1,298 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {
+    InteractionManager,
+    View,
+    ScrollView,
+    ViewPagerAndroid,
+    Platform,
+    StyleSheet
+} from 'react-native';
+
+export default class Swiper extends PureComponent {
+    static propTypes = {
+        children: PropTypes.node.isRequired,
+        style: PropTypes.object,
+        showsPagination: PropTypes.bool,
+        paginationStyle: PropTypes.object,
+        initialPage: PropTypes.number,
+        width: PropTypes.number,
+        dotColor: PropTypes.string,
+        activeDotColor: PropTypes.string,
+        keyboardShouldPersistTaps: PropTypes.string,
+        onIndexChanged: PropTypes.func
+    };
+
+    static defaultProps = {
+        removeClippedSubviews: true,
```

Comment:
You've added this to the ScrollView below. Unless we're planning on making this configurable, I don't think it needs to be a prop